app/views/performance/index.html.haml

```diff
@@ -0,0 +1,23 @@
+- content_for :page_title, "Performance"
+
+%h1= yield :page_title
+
+%h2 Comments on planning applications
+
+- if @comments.any?
+  %table
+    %caption
+      New and returning people commenting on applications over time
+    %thead
+      %tr
+        %th Date
+        %th First time commenters
+        %th Returning commenters
+    %tbody
+      - @comments_metrics_timespan.each do |day|
```

Comment:
I've moved some of this logic into the controller, are their any other refactors you can see here @henare ?